### DISCUSSION TITLE
Can anyone please help me with someone who has hacked my Google account through Ransomware?

### DISCUSSION THREAD

**Discussion by Skol619** [community/community#81524] [New to GitHub]
Can anyone please help me delete third party apps on my android phones that I cannot delete on my android phone?

**Comment by queenofcorgis** [comment]
Hi @Skol619 , Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support can only provide support for GitHub itself so we are going to close this post.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!
